Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb

```diff
@@ -0,0 +1,503 @@
+    "    m_stars_gmr = [+1.4, +1.4]\n",
+    "    m_stars_rmi = [+0.5, +1.5]\n",
+    "\n",
+    "    other_stars_gmr = [-0.5, +1.4]\n",
+    "    other_stars_rmi = [-0.4, +0.5]\n",
+    "\n",
+    "    model_gmr = other_stars_gmr + m_stars_gmr\n",
+    "    model_rmi = other_stars_rmi + m_stars_rmi\n",
+    "    \n",
+    "    return model_gmr, model_rmi\n",
+    "\n",
+    "    \n",
+    "def get_stellar_locus_davenport(color1='gmr', color2='rmi',\n",
+    "                                datafile='Davenport_2014_MNRAS_440_3430_table1.txt'):\n",
+    "    data = pd.read_table(datafile, sep='\\s+', header=1)\n",
+    "    return data[color1], data[color2]\n",
+    "    \n",
+    "    \n",
+    "def plot_stellar_locus(color1='gmr', color2='rmi',\n",
+    "                       color='red', linestyle='--', linewidth=2.5):\n",
+    "    model_gmr, model_rmi = get_stellar_locus_davenport(color1, color2)\n",
+    "    plot_kwargs = {'linestyle': linestyle, 'linewidth': linewidth, 'color': color,\n",
+    "                   'scalex': False, 'scaley': False}\n",
+    "    plt.plot(model_gmr, model_rmi, **plot_kwargs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "def plot_color_color(z, color1, color2, range1=(-1, +2), range2=(-1, +2), bins=31):\n",
+    "    \"\"\"Plot a color-color diagram.  Overlay stellar locus\"\"\"\n",
+    "    band1, band2 = color1[0], color1[-1]\n",
+    "    band3, band4 = color2[0], color2[-1]\n",
+    "    H, xedges, yedges = np.histogram2d(\n",
+    "        z['%s_mag' % band1] - z['%s_mag' % band2],\n",
+    "        z['%s_mag' % band3] - z['%s_mag' % band4],\n",
+    "        range=(range1, range2), bins=bins)\n",
+    "        \n",
+    "    zi = H.T\n",
+    "    xi = (xedges[1:] + xedges[:-1])/2\n",
+    "    yi = (yedges[1:] + yedges[:-1])/2\n",
+    "\n",
+    "    plt.figure(figsize=(8, 8))\n",
+    "    plt.pcolormesh(xi, yi, zi, cmap=cmap)\n",
+    "    plt.contour(xi, yi, zi)\n",
+    "    plt.xlabel('%s-%s' % (band1, band2))\n",
+    "    plt.ylabel('%s-%s' % (band3, band4))\n",
+    "\n",
+    "    plot_stellar_locus(color1, color2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbpresent": {
+     "id": "3a89fe4d-718f-4748-9ca7-28e0b805e04d"
+    },
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(8,6.5))\n",
+    "plt.hist2d(df['g_mag']-df['r_mag'], df['r_mag']-df['i_mag'],\n",
+    "           cmap=cmap, range=((-1, +2), (-1, +2)), bins=51)\n",
+    "plt.colorbar()\n",
+    "plt.xlabel('g-r')\n",
+    "plt.ylabel('r-i')\n",
+    "\n",
+    "plot_stellar_locus('gmr', 'rmi')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Clean this up a bit to only include stars with SNR > 3 in g and r"
```

Comment:
You say you are going to implement a cut for SNR > 3, but then impliment 3 different cutoffs. This left me with some minor but unnecessary confusion. Explain first that you are going to use different SNR to create 3 different catagories: good snr, bright stars, and galaxies.

Reply:
👍